package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>cyclonedds</name>
+    <version>0.1.0</version>
+    <description>Eclipse Cyclone DDS is a very performant and robust open-source DDS implementation. Cyclone DDS is developed completely in the open as an Eclipse IoT project.</description>
+    <maintainer email="cyclonedds-dev@eclipse.org">Eclipse Foundation, Inc.</maintainer>
+    <license>Eclipse Public License 2.0</license>
+    <url type="website">https://projects.eclipse.org/projects/iot.cyclonedds</url>
```

Comment:
It would be good to also add a bugtracker and repository URL: like https://github.com/ros/catkin/blob/461cd3c217f952f908a3cba56927660764b9ae62/package.xml#L13-L14